ui/src/styles.css

```diff
@@ -312,17 +312,6 @@ form .form-block > label,
   font-weight: 300;
 }
 
-/*
- * Force the Zendesk web widget "launcher" button to be hidden at all times.
- * This is a style-based workaround to ensure the Zendesk launcher button isn't
- * shown. Ideally, hiding the launcher would be possible via the Zendesk web widget
- * API, but certain API limitations don't make that possible. See the AoU Zendesk
```

Comment:
That doc is here -> https://docs.google.com/document/d/1rbLAClEsHXlCWl1EmD7dEjeolfrrl_4v7NnUXwciP4s/edit?tab=t.0